scout/adapter/mongo/omics_variant.py

```diff
@@ -1,11 +1,14 @@
 import logging
 from typing import Dict, Optional
 
+from pymongo import ASCENDING, DESCENDING
+
 from scout.constants import OMICS_FILE_TYPE_MAP
 from scout.models.omics_variant import OmicsVariantLoader
 from scout.parse.omics_variant import parse_omics_file
 
 LOG = logging.getLogger(__name__)
+SORTING_ORDER = {"asc": ASCENDING, "desc": DESCENDING}
```

Comment:
```suggestion
SORT_ORDER = {"asc": ASCENDING, "desc": DESCENDING}
```

Reply:
Unless you feel strongly about it, that's what we usually call it. 😊 It mostly seems a difference in perspective to me, so whatever, but every time you read about it in algorithms it would be called "sort order".

Reply:
SORT_ORDER is fine, I don't care about the name!